### DISCUSSION TITLE
Auto Pause Does Not Work Galaxy 21 Ultra

### DISCUSSION THREAD

**Discussion by dopeboyglory** [d4rken-org/capod#267] [Q&A]
This feature does not work. I upgraded to premium very disappointed that its not working 😕.

**Comment by baelrayneVessdren** [comment]
Had the same issue. Had to force stop the app a few times to get it to work. 

In my App settings > Reactions > Deactivated One pod mode and activated everything else. Took both pods out and force stopped the app. Re-started it, put pods in my ears and waited till the app showed me "In ear" for both. Took one out after and had to wait ~2 seconds for the music to pause.
